packages/forge2d/test/dynamics/joints/rope_joint_def_test.dart

```diff
@@ -5,6 +5,8 @@ void main() {
   group('RopeJointDef', () {
     test('can be instantiated', () {
       expect(RopeJointDef(), isA<RopeJointDef>());
+      expect(RopeJointDef().localAnchorA, equals(Vector2.zero()));
+      expect(RopeJointDef().localAnchorB, equals(Vector2.zero()));
     });
```

Comment:
I think this should be on another test.

Could you create a test that checks "sets localAnchorA to zero" and a test named "sets localAnchorB to zero" when instantiated?

Reply:
Sure thing!